src/Message/UpdatePaymentMethodRequest.php

```diff
@@ -28,11 +27,12 @@ public function getData()
      * Send the request with specified data
      *
      * @param  mixed $data The data to send
+     *
      * @return ResponseInterface
      */
     public function sendData($data)
     {
-        $response = $this->braintree->paymentMethod()->update($data['token'], $data['options']);
+        $response = $this->braintree->paymentMethod()->update($data['token'], $data);

```

Comment:
Is this change related to the v3 upgrade?

Reply:
No, this was a bug in previous version. I was fixed there on my fork when using with v2.x and it is still actual.

Reply:
This can be changed to `$data['options'] ?? []` as well

Reply:
I just tested with `$data['options'] ?? []` instead of `data`, it is not working because of `Braintree\Util`'s validation. So the only way it will work is the one I've commited